src/esp/assets/GenericInstanceMeshData.cpp

```diff
@@ -262,6 +265,15 @@ void GenericInstanceMeshData::uploadBuffersToGPU(bool forceReload) {
       .setIndexBuffer(std::move(indices), 0,
                       Mn::GL::MeshIndexType::UnsignedInt);
 
+  cpu_ibo_ = std::move(new_indices);
+
+  collisionMeshData_.positions =
```

Comment:
Since this is duplicated from line 215, maybe move this into a helper?

Reply:
Could be also just this, I think:

```cpp
collisionMeshData_.positions = Cr::Containers::arrayCast<Mn::Vector3>(Cr::Containers::arrayView(cpu_vbo_));
collisionMeshData_.indices = Cr::Containers::arrayCast<Mn::UnsignedInt>(Cr::Containers::arrayView(cpu_ibo_));
```